.gitignore

```diff
@@ -203,4 +203,9 @@ FakesAssemblies/
 /tools/*.dll
 *.GhostDoc.xml
 pingme.txt
-groupMapping*.json
\ No newline at end of file
+groupMapping*.json
```

Comment:
these additions to the gitignore are incorrect.  Please revert changes to this file unless you have  a file you would liek to add.